ui/src/app/utils/runtime-utils.tsx

```diff
@@ -681,7 +681,6 @@ export const withAnalysisConfigDefaults = (
   let existingDiskName = null;
   const computeType = r.computeType ?? ComputeType.Standard;
   if (computeType === ComputeType.Standard) {
-    dataprocConfig = null;
```

Comment:
this looks safe because it is always set below here.  It *also* looks like we never use the value passed in to the function, so we should remove that from line 677

Reply:
dataprocConfig  is used in Dataproc check L787